learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+| relevancyStrictness | Not Supported |
+| maxValuesPerFacet | Not Supported |
+| sortFacetValuesBy | Not Supported |
+| restrictHighlightAndSnippetArrays | Not Supported |
+
+## APIs
+
+This section compares Algolia and Meilisearch's respective API methods, using JavaScript for reference.
+
+| Topic | Algolia | Meilisearch |
+| --- | --- | --- |
+| Index Instantiation | `client.initIndex()`<br>Here, client is an Algolia instance. | `client.index()`<br>Here, client is a Meilisearch instance. |
+| Create Index | Users don’t need to create an index explicitly, the engine does it for you the first time you add an object or set settings. | The same applies for Meilisearch but users can also create an index explicitly using: `client.createIndex(string indexName)` |
+| Get All Indexes | `client.listIndices()` | `client.getIndexes()` |
+| Get Single Index | No method available | `client.getIndex(string indexName)` |
+| Delete Index | `index.delete()` | `client.deleteIndex(string indexName)` |
+| Get Index Settings | `index.getSettings()` | `index().getSettings()` |
+| Update Index Settings | `index.setSettings(object settings)` | `index().updateSettings(object settings)` |
+| Search Method | `index.search(string query, { searchParameters, requestOptions })` | `index.search(string query, object searchParameters)` |
+| Add Object | `index.saveObjects(array objects)` | `index.addDocuments(array objects)` |
+| Partial Update Object | `index.partialUpdateObjects(array objects)` | `index.updateDocuments(array objects)` |
+| Delete All Objects | `index.deleteObjects(array objectIDs)` | `index.deleteAllDocuments()` |
+| Delete One Object | `index.deleteObject(string objectID)` | `index.deleteDocument(string id)` |
+| Get All Objects | `index.getObjects(array objectIDs)` | `index.getDocuments(object params)` |
+| Get Single Object | `index.getObject(str objectID)` | `index.getDocument(string id)` |
+| Get API Keys | `client.listApiKeys()` | `client.getKeys()` |
+| Get API Key Info | `client.getApiKey(string apiKey)` | `client.getKey(string apiKey)` |
+| Create API Key | `client.addApiKey(array acl)` | `client.createKey(object configuration)` |
+| Update API Key | `client.updateApiKey(string apiKey, object configuration)` | `client.updateKey(string apiKey, object configuration)` |
+| Delete API Key | `client.deleteApiKey(string apiKey)` | `client.deleteKey(string apiKey)` |
+
+## Front-end components
+
+[InstantSearch](https://github.com/algolia/instantsearch.js) is a collection of open-source tools maintained by Algolia and used to generate front-end search UI components.
+
+[Instant Meilisearch](https://github.com/meilisearch/instant-meilisearch) is a plugin connecting your Meilisearch instance with InstantSearch, giving you access to many (but not all) of the same front-end components as Algolia users. Here is [an up-to-date list of components compatible with Instant Meilisearch](https://github.com/meilisearch/instant-meilisearch/#-api-resources).
```

Comment:
I would open this paragraph by saying something like "To use InstantSearch with Meilisearch, you need Instant Meilisearch" and so on.

Might be me, but I feel that right now the tools are presented as equivalent, when actually one depends on the other.